pkgs/applications/networking/mailreaders/meli/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchgit, rustPlatform, pkgconfig, openssl, dbus }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "meli";
+  version = "pre-alpha-0.3.0";
+
+  src = fetchgit {
+    url = "https://git.meli.delivery/meli/meli.git";
+    rev = "${version}";
+    sha256 = "0slg9gz3800lxhfdqwlzqk3j9yi2yq7yl53dcyj6vhzk6kgz8lc2";
+  };
+  cargoPatches = [ ./cargo-lock.patch ];
```

Comment:
An extra newline above this one would be nice for consistency

Reply:
Updated.